src/components/Form.tsx

```diff
@@ -0,0 +1,275 @@
+import { CHECK_AND_RADIO, RANGE } from "../data";
+import Alert from "./Alert";
+import React, { useState } from "react";
+import {
+  Box,
+  Grid,
+  Slider,
+  Input,
+  OutlinedInput,
+  InputLabel,
+  MenuItem,
+  FormControl,
+  Select,
+  Chip,
+  Button,
+  Stack,
+  Typography,
+} from "@mui/material";
+import { SelectChangeEvent } from "@mui/material/Select";
+import ClearIcon from "@mui/icons-material/Clear";
+import SendIcon from "@mui/icons-material/Send";
+
+export interface AI {
+  levelOfAI: string[];
+  whereAIIsUsed: string[];
+  TypeOfAI: string;
+  rateAIIntelligence: number;
+}
```

Comment:
looks like this is simmilar to `export interface ICardData` maybe it make sense not to duplicate this parts?